src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private Random random = new Random();
+    private int size1 = random.nextInt(10 + 1);
+    private int size2 = random.nextInt(10 + 1);
+    private int size3 = random.nextInt(10 + 1);

```

Comment:
These variables are not needed then random numbers will only be generated once.  Pass their values directly to the constructor

Reply:
My comment is ignored

Reply:
![image](https://user-images.githubusercontent.com/118083945/205447634-955614a2-b5d8-4092-a12f-74e5260dc79d.png)
 mvn says that line too long... if i put it inside parameters...

Reply:
![image](https://user-images.githubusercontent.com/118083945/205447718-dbd332c2-2d96-40c7-9c35-c3f5b448cc1c.png)